app/helpers/templateGenerator.js

```diff
@@ -129,17 +129,21 @@ const generateBlock = (params, blockType) => (
   })).join('')
 );
 
-const getSettings = (name, type) => {
-  const settingsString = fs.readFileSync(`app/${type}s/${name}/settings.json`, 'utf-8');
+const getSettings = (name, type, subtype) => {
```

Comment:
can we rename this now so `type` is `templateType` and `subType` is `componentType`